website/docs/d/pi_network_interfaces.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_pi_network_interfaces"
```

Comment:
```suggestion
page_title: "IBM : pi_network_interfaces"
```

Reply:
@yussufsh Shouldn't this be same as the title? `# ibm_pi_network_interface`
That is what other services do from taking a look at most of them.